Makefile

```diff
@@ -26,14 +26,23 @@ STATIC_CHECK ?= false
 COMPILERS = gcc clang
 FORTIFY_LEVELS = 1 2
 
+RESULTFILE = fortify.result
+
 TARGETS=memcpy memmove mempcpy memset snprintf sprintf stpcpy stpncpy strcat \
 	strcpy strncat strncpy vsnprintf vsprintf
 
 check:$(patsubst %,check-%,$(COMPILERS)) test_common.c
 
+$(RESULTFILE):
+	$(Q)cat /dev/null > $@
```

Comment:
What an interesting idiom. I assume it's equivalent to `rm -f $@ && touch $@` ?

Reply:
On a more general note, you're using `RESULTFILE` to accumulate the result of checks, which may interact surprisingly if some runs under `make -j`. May I suggest to use a result file per run and `grep` through all of them?

Reply:
> What an interesting idiom. I assume it's equivalent to `rm -f $@ && touch $@` ?

Kinda, the `cat` will retain the inode.

Reply:
> On a more general note, you're using `RESULTFILE` to accumulate the result of checks, which may interact surprisingly if some runs under `make -j`. May I suggest to use a result file per run and `grep` through all of them?

Good point; I'll split out the results and consolidate in the end.